### DISCUSSION TITLE
Book navigation : improve navigation between left and right navbars

### DISCUSSION THREAD

**Discussion by wilcar** [quarto-dev/quarto-cli#2759] [Q&A]
By default, quarto allows to navigate between each page i.e. each qmd file with arrows and displays the level 1 titles on the left navbar and the level 2 or level 3 titles in the right navigation bar.
I would like to have the navigation arrows for each level 2 titles . 

Two possibilities ?
 
- For a single qmd file: is it possible to enable navigation arrows for level 2 titles to avoid long scrolls in pages.

- For multiple qmd files corresponding to chapters :
eg: 

- part: cards.qmd
    chapters:
      - objects.qmd
      - notation.qmd
      - modifying.qmd
      - environments.qmd

Is it possible to keep the navigation between parts in the left menu and chapters navigation in the right bar?

**Comment by dragonstyle** [comment]
We are currently limited in what books support- basically parts / chapters in the sidebar on the left, and the contents of the current chapter (sections) on the right. We're planning to do some work on books in an upcoming release to try make them more flexible and will consider this then!
